src/controllers/controller.bar.js

```diff
@@ -181,6 +181,10 @@ function parseArrayOrPrimitive(meta, data, start, count) {
 	return parsed;
 }
 
+function isFloatBar(custom) {
+	return custom && custom.barStart !== undefined && custom.barEnd !== undefined;
```

Comment:
here we check `custom.barStart` and `custom.barEnd` but line 260 checks `start` and `end`. Is that correct?

Reply:
Not strictly, but at least its consistent now. Old line 255 checked for just `custom` and if defined, used start and end from it.

We could add `floatBar: true` to line 150 and use that instead.

Reply:
Just wanted to make sure it was expected. I see the logic here is used further down as well